kolibri/core/assets/src/styles/themeSpec.js

```diff
@@ -42,6 +43,25 @@ const _imageSpec = {
 };
 
 export default {
+  appBar: {
+    type: Object,
+    default: null,
+    spec: {
+      background: {
+        type: String,
+        default: themeTokens().appBar,
```

Comment:
We don't need to update it here - but if we are going to remove it from the themeTokens in KDS, might just want this to map to something more directly.